### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#457] [Esami & Esoneri M-Z]
## Traccia

<img width="678" alt="image" src="https://github.com/user-attachments/assets/d3a59e82-c78f-4a2f-a30e-23b6a658de43">

**Comment by luckignolo32** [comment]
<img width="930" alt="{903993E6-8846-477C-9DDF-37AA20D6D415}" src="https://github.com/user-attachments/assets/eaeb4187-e6b9-4edc-a823-171d3a0b7eec">
<img width="925" alt="{F1280B21-9BB5-4C52-95E9-CA32BE3DEBC9}" src="https://github.com/user-attachments/assets/87f7510c-d791-47d1-b817-a4069cd9c898">

**Comment by rimaout** [comment]
<img width="868" alt="image" src="https://github.com/user-attachments/assets/513944ec-968a-4c49-8839-7ab94405ab8f">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/b2a1fcf4-6501-4770-b7c0-1cbbb1341964)
